input/pagecontent/experimental.md

```diff
@@ -0,0 +1,102 @@
+{% endcomment %} -->
+<!-- experimental.md {% comment %}
+*****************************************************************************************
+*                            WARNING: DO NOT EDIT THIS FILE                             *
+*                                                                                       *
+* This file is generated by SUSHI. Any edits you make to this file will be overwritten. *
+*                                                                                       *
+* To change the contents of this file, edit the original source file at:                *
+* fhir-mCODE-ig\input\pagecontent\experimental.md                                       *
+*****************************************************************************************
+{% endcomment %} -->
+<!-- experimental.md {% comment %}
+*****************************************************************************************
+*                            WARNING: DO NOT EDIT THIS FILE                             *
+*                                                                                       *
+* This file is generated by SUSHI. Any edits you make to this file will be overwritten. *
+*                                                                                       *
+* To change the contents of this file, edit the original source file at:                *
+* fhir-mCODE-ig\input\pagecontent\experimental.md                                       *
+*****************************************************************************************
+{% endcomment %} -->
+The profiles and value sets on this page are experimental.
+
+### Tumor size
+
+This is a set of experimental profiles for representing the dimensions of a tumor.
+
+| Type     | Name               | Description                   | Example             |
+| -------- | ------------------ | ----------------------------- | ------------------- |
+| Profile  | [TumorSize]        | Defines dimensions of a tumor | [TumorSize example] |
+| Profile  | [Tumor]            | Identifies a specific tumor   | [Tumor example]     |
+| ValueSet | [TumorSizeUnitsVS] | Limits units to `mm` or `cm`  | n/a                 |
+
+[TumorSize]: StructureDefinition-mcode-tumor-size.html
+[TumorSize example]: Observation-mcode-tumor-size.html
+[Tumor]: StructureDefinition-mcode-tumor.html
+[Tumor example]: BodyStructure-mcode-tumor.html
+[TumorSizeUnitsVS]: ValueSet-mcode-tumor-size-units-vs.html
+
+#### Usage notes
+
+- If the tumor has been removed from the body, use `specimen` to reference a [Specimen](http://hl7.org/fhir/R4/specimen.html) resource. If the tumor is still in the body, use `focus` to reference a resource confirming to the [Tumor] profile.
+- The tumor dimensions are stored in three different components in [TumorSize]. `tumorLongestDimension` is required, and `tumorDimension2` and `tumorDimension3` are optional.
```

Comment:
This is the only part of the experimental tumor size profile I saw that snuck in. This entire file can be safely deleted...it’s in the experimental tumor size branch as well.